Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +137,32 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (array == null || array.Length == 0 || index < 0 || index > array.Length - 1)
+            {
+                return array;
+            }
+
+            int[] newArray = new int[array.Length - 1];
+            bool isAfterIndex = false;
+
+            for (int i = 0; i < array.Length - 1; i++)
+            {
+                if (i == index)
+                {
+                    isAfterIndex = true;
+                }
+
+                if (!isAfterIndex)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886613)